### DISCUSSION TITLE
Endpoints on the sidebar should display the HTTP method

### DISCUSSION THREAD

**Discussion by thelinuxlich** [scalar/scalar#360] [Ideas]
Just like the search modal, the sidebar should display the HTTP methods, in my screenshot I have two endpoints GET/POST and can't differentiate them without opening 
![image](https://github.com/scalar/scalar/assets/5043/f43bf906-bc81-431a-8326-3d72ce31b8fa)

**Comment by marclave** [comment]
Appreciate the feedback!

@cameronrohani, @hanspagel and I are going to have a think on this one.. We can do something nice :) Might need to do it a little bit differently than the sidebar but will update once we have something ✨

**Comment by marclave** [comment]
@thelinuxlich check it out in this PR https://github.com/scalar/scalar/pull/365 ! :)
